osv/impact.py

```diff
@@ -322,17 +328,17 @@ def _batcher(entries, batch_size):
     yield entries[i:i + batch_size], i + batch_size >= len(entries)
 
 
-def _throttled_put(to_put):
+def _throttled_put(to_put, batch_size=_DATASTORE_BATCH_SIZE):
```

Comment:
nit: Prefer removing the defaults and explicitly setting the batch_size variable. `_throttled_put` and `_throttled_delete` are only used once each with the default value, so only two more line changes. 

 This avoids potential problem in the future where someone uses throttled_put assuming it will be throttling to `_DATASTORE_LARGE_BATCH_SIZE = 8`, but instead is throttling a lot less than that.

Reply:
Hmm good point, I'm going to delete the other two uses completely very soon after the migration is done. I'll clean this up then!